File: src/main/java/com/tom/Hello.java
package com.tom;

import java.sql.SQLOutput;

public class Hello {
    public static void main(String[] args) {
        //System.out.println("Hello");
     Person p=new Person();
     p.hello();
     p.weight=66.5f;
     p.height=1.7f;
     System.out.println(p.bmi());
     /*int age=19;
     char c='x';
     Character c2='A';
     byte b=120;
     float weight =66.5f;
     double heighrt=1.7;
     boolean adult=true;
     String name="tom";*/

    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main() {
    //println("Hello")
    val h = Human(weight = 66.5f,height = 1.7f)
    println(h.bmi())
    val score=88
    println(score>60)
    val c:Char='A'
    println(c.toInt()>60)
    /*var age = 19
    age = 20
    var weight = 66.5f
    var name: String;
    name = "cc"*/
}
class Human(var name:String="",var weight: Float,var height:Float) {
    init{
        println("test $weight")
    }
    //constructor(name:String,weight: Float,height: Float)
    fun bmi():Float{
    val bmi =weight/(height*height)
        return bmi}
    /*fun hello() {
        println("Hello K")
    }*/
}